tools/downloader/common.py

```diff
@@ -328,13 +329,15 @@ def deserialize(cls, model, name, subdirectory):
                 with deserialization_context('"postprocessing" #{}'.format(i)):
                     postprocessing.append(Postproc.deserialize(postproc))
 
-            pytorch_to_onnx_args = None
-            if model.get('pytorch_to_onnx', None):
-                pytorch_to_onnx_args = [validate_string('"pytorch_to_onnx" #{}'.format(i), arg)
-                                        for i, arg in enumerate(model['pytorch_to_onnx'])]
-
             framework = validate_string_enum('"framework"', model['framework'], KNOWN_FRAMEWORKS)
 
+            conversion_to_onnx_args = None
+            if model.get('conversion_to_onnx_args', None):
+                if framework not in KNOWN_ONNX_SUPPORT:
+                    raise DeserializationError('Conversion to ONNX not supported for "{}" framework!'.format(framework))
```

Comment:
Please remove the exclamation mark, it goes against the error message convention.